### ISSUE TITLE
Multiple Log View Locations

### ISSUE THREAD

**Issue by Nowi5** [rap2hpoutre/laravel-log-viewer#162]
Hi all,

I hope you are doing fine. Do we have any option to define multiple locations e.g. via config setting storage_path to array = ['/location1', '/other/location2']?

Thank you.

**Comment by elminson** [comment]
Hi, @Nowi5 sounds really good, let me see if I can come with something!

**Comment by Nowi5** [comment]
Thank you very much. And here is the reason why:
https://laravel-tenancy.com/docs/hyn/5.2/logging

'storage_path' => [env('LOGVIEWER_STORAGE_PATH', 'logs'), 'storage/app/tenancy/tenants']

Maybe it is enough to come up with another pattern for the storage_path and searching sub folders automatically.

**Comment by elminson** [comment]
Hello  @Nowi5, I just post a PR  #161 https://github.com/rap2hpoutre/laravel-log-viewer/pull/161 (@rap2hpoutre  have to review first) that allow multiple log files locations 
now you can use 
```php
    'storage_path' => [env('LOGVIEWER_STORAGE_PATH', 'logs'),'storage/app/tenancy/tenants'],
```

**Comment by Nowi5** [comment]
Wow... that is awesome. Thank you very much! Will have a look soon!

**Comment by rap2hpoutre** [comment]
@elminson PR #161 is about reducing class complexity, not multitenancy, did I missed something?

**Comment by Nowi5** [comment]
I would guess that the multiple config files setting is a nice side effect ;-)

**Comment by elminson** [comment]
@Nowi5 you are right :) I will try to create a PR for the multiple config files settings, do you agree @rap2hpoutre ?

**Comment by elminson** [comment]
Clean PR Created @Nowi5 @rap2hpoutre  💯

**Comment by rap2hpoutre** [comment]
@elminson Thank you!

@Nowi5 Fixed, available in v1.1.0
